projects/components/src/sequence/renderer/sequence-bar-renderer.service.ts

```diff
@@ -230,7 +230,7 @@ export class SequenceBarRendererService {
               ? scaledEnd - this.markerWidth - 2
               : marker.markerTime
         });
-        markerTime = scaledNormalizedMarkers[index].markerTime;
```

Comment:
why do we not need normalized markers?

Reply:
This is just the terminology I have used, 
No other significance of it than grouping the scaled markers
If you could suggest the better naming for this?